community-members.md

```diff
@@ -32,14 +32,35 @@ These are community maintainers, responsible for cross-functional project commun
 
 [David Hirsch](https://github.com/DavidPHirsch) Dynatrace
 
-## Specifications and Approvers
+### Project roles
+
+The following project roles are defined at the moment:
+_Community Participant_,
+_Contributor_,
+_Organization Member_,
+_Triager_,
+_Approver_,
+_Maintainer_,
+_Technical Committee Member_,
+_Bootstrap Governance Committee Member_.
+These roles are defined below.
+
+See [Contributor Ladder](./CONTRIBUTOR_LADDER.md) for additional information.
+
+In addition to the elected maintainer roles,
+3 individuals get the maintainer status at the inception of the project:
```

Comment:
Do you know if this section is here for historical purposes, or does maintainer status always apply to the listed individuals?